### DISCUSSION TITLE
Understanding What To Install From Android Studio

### DISCUSSION THREAD

**Discussion by dmunsie** [airsdk/Adobe-Runtime-Support#3004] [Q&A] [answered]
I always get confused with this, so I'd like a clear path of understanding this.

When it comes to installing SDK Platforms and SDK Tools, should we always install the latest versions? In the airsdk.dev/docs/basics/install/windows it says "Start Android studio and run through the setup wizard **which will install the latest Android SDK**, command line tools and build tools."

If there were specific versions required, would the install guide be updated with that information or would be it made available somewhere else? Thank you for any replies.

**Comment by ajwfrost** [comment, marked as answer]
Generally, yes, it's always worth having the latest versions of everything. The only time I would have said not is around the use of Java, when Oracle started hiding the 'sun.security' classes which meant that we needed to stick with JDK16 or earlier. But with our latest release, we've dropped those dependencies so you can use Java 17 and later..

From Android, we mostly use the command line (platform) tools, and the build tools; the actual SDK/API set is something we use in our own build process but isn't used in the APK/AAB packaging. We also use Gradle and the Android Gradle Plug-in, but that's generally configured in our SDKs or can be overridden by the project if needed.

On macOS they always force everyone to update to the latest versions/tools as soon as possible :-)

**Reply by dmunsie** [reply]
Sounds good, thanks for the reply. Might be good idea on the https://airsdk.harman.com/download page to list the current recommended install files to use with the latest air sdk. Appreciate all your work!
